data/856/325/81/85632581.geojson

```diff
@@ -1180,5 +1249,5 @@
388,27.824139],[34.265778,27.822861],[34.266666,27.822861],[34.270363,27.818306],[34.27039,27.815777],[34.266224,27.81164],[34.265415,27.808306],[34.264557,27.807528],[34.264526,27.799166],[34.263748,27.798306],[34.263695,27.793306],[34.262917,27.792473],[34.26289,27.790001],[34.261639,27.78875],[34.259945,27.787916],[34.253307,27.787056],[34.252529,27.786194],[34.250805,27.787083],[34.247528,27.787916],[34.241695,27.785389],[34.237915,27.788334],[34.237862,27.790001],[34.235832,27.792084],[34.232498,27.792917],[34.22747,27.787056],[34.221668,27.787111],[34.220001,27.787861],[34.217888,27.790001],[34.216251,27.793333],[34.214138,27.793751],[34.212917,27.789139],[34.210835,27.787889],[34.208332,27.787916],[34.206249,27.785805],[34.207027,27.78161],[34.211224,27.775833],[34.214554,27.773306],[34.214527,27.772499],[34.219139,27.767916],[34.223305,27.766222],[34.228333,27.766251],[34.234165,27.763721],[34.236668,27.76375],[34.237446,27.762888],[34.239944,27.762917],[34.241638,27.763721],[34.243279,27.766277],[34.247471,27.767111],[34.248306,27.767889],[34.254112,27.767084],[34.256222,27.764944],[34.252083,27.759138],[34.252083,27.752501],[34.25539,27.74914],[34.257084,27.745806],[34.257057,27.738306],[34.26125,27.733334],[34.261223,27.730806],[34.25872,27.729973],[34.259609,27.729166],[34.259167,27.725389],[34.2575,27.725389],[34.253277,27.727888],[34.25375,27.724972],[34.252472,27.724583],[34.251667,27.722918],[34.245804,27.722918],[34.242054,27.725805],[34.242054,27.728333],[34.243305,27.729528],[34.244972,27.728722],[34.245361,27.729166],[34.242863,27.730862],[34.239971,27.735416],[34.235001,27.73539],[34.233276,27.736279],[34.232918,27.739944],[34.230835,27.742027],[34.229973,27.742027],[34.228333,27.744583],[34.227501,27.744583],[34.224583,27.747473],[34.224529,27.74914],[34.22039,27.753305],[34.219582,27.755833],[34.215363,27.76],[34.214554,27.762472],[34.211639,27.766222],[34.2075,27.766251],[34.207085,27.767445],[34.204582,27.769138],[34.203693,27.771667],[34.198723,27.775833],[34.197529,27.777916],[34.195389,27.778278],[34.195415,27.780001],[34.193306,27.782084],[34.188278,27.782055],[34.187527,27.782917],[34.183334,27.782917],[34.182499,27.783722],[34.18,27.783751],[34.177502,27.785418],[34.174973,27.785389],[34.174137,27.784584],[34.165833,27.784555],[34.164112,27.782084],[34.160862,27.783722],[34.159138,27.785418],[34.157444,27.785418],[34.15411,27.787056],[34.151669,27.787083],[34.147499,27.788778],[34.142502,27.791222],[34.140835,27.792889],[34.131611,27.792917],[34.130806,27.793722],[34.127499,27.793751],[34.12661,27.794556],[34.118305,27.794556],[34.117474,27.795389],[34.111668,27.795416],[34.110832,27.794584],[34.104168,27.793777],[34.100807,27.791222],[34.098278,27.791222],[34.095001,27.786278],[34.092056,27.786638],[34.092083,27.790833],[34.090805,27.791222],[34.089138,27.792055],[34.087502,27.794584],[34.085804,27.794611],[34.082474,27.796223],[34.07914,27.798721],[34.077473,27.798721],[34.07164,27.802111],[34.069195,27.802055],[3
```

Comment:
The southern border should be a straight line, let's look at this in QGIS to understand what's going on:

![image](https://user-images.githubusercontent.com/853051/135945993-2855778c-18ef-497a-8ea3-2af8f91da3c9.png)